### PR TITLE
feat: slide 10 of A3, sliding list

### LINKED ISSUE #102: 🚀 Feature: A3: Animation: Slide 10
* [ ] :wave: I may be able to implement this feature request

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by reginawang99**
closes #102

### CODE REVIEW

File: src/components/Activities/Activity3/Animations/List.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    'Video Feedback',
+    'Video Number of Views with respect to Time',
+    'Credibility of Video',
+    'Video Content',
+    'Video Title',
+    'Video Tags',
+    'Video Creator',
+    'Video Variety',
+  ];
+  const MAPPED_PARAMETERS = RECOMMENDATION_PARAMETERS.map((element) => <><p/>{element}</>);
```
Comment (BryanPan342) [inline src/components/Activities/Activity3/Animations/List.tsx:38]: should this just be a <p> tag?

**Review by BryanPan342 — CHANGES_REQUESTED** [review]
minor change

**Comment by reginawang99** [comment]
lint complains `  38:72  error  Missing "key" prop for element in iterator  react/jsx-key` 
hope it doesnt matter :-0

**Review by BryanPan342 — APPROVED** [review]
LGTM

**Pull request merged**